src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -95,6 +100,10 @@ public class ExceptionlessSink : ILogEventSink, IDisposable {
 
             var builder = _client.CreateFromLogEvent(logEvent).AddTags(_defaultTags);
 
+            var minLogLevel = _client.Configuration.Settings.GetMinLogLevel(logEvent.GetSource());
+            if (logEvent.GetLevel() < minLogLevel)
+                return;
```

Comment:
If we move this up before the builder, it will prevent any extra allocations.

```suggestion
            var minLogLevel = _client.Configuration.Settings.GetMinLogLevel(logEvent.GetSource());
            if (logEvent.GetLevel() < minLogLevel)
                return;

            var builder = _client.CreateFromLogEvent(logEvent).AddTags(_defaultTags);
```

Reply:
Yes, modified